draft-ietf-tls-esni.md

```diff
@@ -797,9 +807,12 @@ transmitted on the wire by the client:
 1. If CH1 contains the "encrypted_client_hello" extension but CH2 does not, or
    if CH2 contains the "encrypted_client_hello" extension but CH1 does not, then
    the server MUST abort the handshake with an "illegal_parameter" alert.
-1. Suppose the "encrypted_client_hello" extension is sent in both CH1 and CH2,
-   If the configuration identifier (see {{ech-configuration}}) differs between
-   CH1 and CH2, then the server MUST abort with an "illegal_parameter" alert.
+1. If the "encrypted_client_hello" extension is sent in both CH1 and CH2,
+   the server follows the procedure in {{client-facing-server}} to decrypt the
+   second ClientHelloOuter, but it uses the previously-selected ECHConfig as
```

Comment:
s/ClientHelloOuter/ClientHelloInner/?

Reply:
Hah! I meant to write "decrypt the second ClientHelloOuter [into the second ClientHelloInner]", but I guess you more naturally read it as "decrypt the second ClientHelloInner [from of the second ClientHelloOuter]". Which means I should probably find a different wording all together. Will wordsmith.

Reply:
How's this new version?

Reply:
LGTM!